scripts/cp-pkgs.js

```diff
@@ -77,9 +90,24 @@ function cpAsset(asset) {
   return cpFile(asset, destDir).then(() => console.log(`cp ${asset} -> ${destDir}`));
 }
 
+function cpDeclarationAsset(asset) {
+  const assetPkg = path.parse(asset.split('build/')[1]).dir;
+  if (!fs.existsSync(assetPkg)) {
+    Promise.reject(new Error(`Non-existent asset package path ${assetPkg} for ${asset}`));
+  }
+  const destFileName = toCamelCase(path.parse(asset).name.replace(/^mdc-|\.d$/gm, ''));
+  const destDir = path.join(assetPkg, 'dist', `mdc.${destFileName}.d.ts`);
+  return cpFile(asset, destDir).then(() => console.log(`cp ${asset} -> ${destDir}`));
```

Comment:
We should probably handle the exception case here too, and at least print out the error